samples/bluetooth/peripheral_hids_keyboard/prj.conf

```diff
@@ -0,0 +1,40 @@
+CONFIG_BT=y
+CONFIG_BT_DEBUG_LOG=y
+CONFIG_BT_MAX_CONN=2
+CONFIG_BT_MAX_PAIRED=2
+CONFIG_BT_SMP=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_DEVICE_NAME="NCS HIDS keyboard"
+CONFIG_BT_DEVICE_APPEARANCE=961
+
+CONFIG_BT_GATT_HIDS=y
+CONFIG_BT_GATT_HIDS_MAX_CLIENT_COUNT=2
+CONFIG_NRF_BT_UUID_16_POOL_SIZE=40
+CONFIG_NRF_BT_CHRC_POOL_SIZE=20
+CONFIG_NRF_BT_CCC_POOL_SIZE=10
+
+CONFIG_BT_LINK_CTX_MANAGER=y
+
+CONFIG_ASSERT=y
+CONFIG_ASSERT_LEVEL=2
+
+CONFIG_BT_GATT_DIS=y
+CONFIG_BT_GATT_DIS_PNP=y
+CONFIG_BT_GATT_DIS_MANUF="NordicSemiconductor"
+CONFIG_BT_GATT_DIS_PNP_VID_SRC=2
+CONFIG_BT_GATT_DIS_PNP_VID=0x1915
+CONFIG_BT_GATT_DIS_PNP_PID=0xEEEF
+CONFIG_BT_GATT_DIS_PNP_VER=0x0100
+
+CONFIG_SYS_LOG=y
+CONFIG_SYS_LOG_DEFAULT_LEVEL=2
+
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+CONFIG_BT_SETTINGS=y
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
```

Comment:
This shouldn't be necessary.

Reply:
all three of them?

Reply:
It is taken from peripheral_hids from zephyr samples. So I am not sure if we should remove it or not.

Reply:
Ok, not an issue.